submissions/Lu-sure/Popup/style.css

```diff
@@ -0,0 +1,219 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: content-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  height: 65px;
+  align-items: center;
+}
+
+.nav-icon, .popup-list {
+  list-style: none;
+}
+
+.nav-icon {
+  display: flex;
+  align-items: center;
+}
+
+.nav-text-item {
+  text-decoration: none;
+  color: #5f5f5f;
+  margin-right: 20px;
+  border: 2px solid transparent;
+  border-radius: 8%;
+  padding: 3px;
+}
+
+
+.nav-img-item, .menu-img {
+  display: block;
+  height: 32px;
+  width: 32px;
+}
+
+.nav-img-item {
+  margin-right: 22px;
+  border: 2px solid transparent;
+  border-radius: 8%;
+  padding: 2px;
+}
+
+.a {
```

Comment:
Try to use more meaningful names for classNames.

Reply:
Thanks, I will pay more attention to this.